tests/inte_test/test_esim.py

```diff
@@ -0,0 +1,118 @@
+import shutil
```

Comment:
`test_models.py` is supposed to test all models, including the newly added ones. As long as the model is imported somewhere while matchzoo's loading, `test_models.py` will find it and test it. It seems there's no special setup for this model, so I guess `auto.prepare` that used by `test_models.py` will just work fine. This means it's better to not add any test file just for ESIM.

Reply:
oh I see, 'cause originally I saw a list of test files for each model under tests/inte_test folder and wondering why they all disappear in the new version... get it now and will remove this file.